### ISSUE TITLE
Detroit - Partners Clean Up 

### ISSUE THREAD

**Issue by kathyccheng** [CityOfDetroit/bloom#1291]
**Rename**
* Leasing Agent Name --> Property Management Company 
* Additional Accessibility --> Additional Accessibility Details (this should be in Partners AND Public) 
  * Public Translations 
    * Spanish - Detalles adicionales de accesibilidad
    * Bengali - অতিরিক্ত অ্যাক্সেসযোগ্যতার বিবরণ
    * Arabic - تفاصيل الوصول الإضافية 
* Community Programs --> Community Types (this should be in Partners AND Public) 
  * Partners 
    * Title Community Types 
    * Description: Tell us about any additional community types related to this listing.
    * Button: Add Community Types
  * Public 
    * Change title to Community Types 
    * Spanish - Tipos de comunidad
    * Bengali - সম্প্রদায়ের ধরন
    * Arabic - أنواع المجتمع

 

![image](https://user-images.githubusercontent.com/3453407/170370067-bd3ce943-44d5-42b5-98fe-81118858dedc.png)
![image](https://user-images.githubusercontent.com/3453407/170370142-3dc747b0-d4b0-468b-9103-6e8a2779b889.png)
![image](https://user-images.githubusercontent.com/3453407/170370524-d8cc51ea-a5aa-472f-813a-a4bf1c9cbd57.png)

**Remove** 
* Accessibility Type in Partners Unit Group panel 
* Is there a referral opportunity 
* Additional Application Submission Notes

![image](https://user-images.githubusercontent.com/3453407/170369716-9ecc632b-0ca3-4673-a693-217a70935a01.png)
![image](https://user-images.githubusercontent.com/3453407/170369885-d8d90210-9214-4730-b1ba-3d8223e42bc2.png)
![image](https://user-images.githubusercontent.com/3453407/170369929-4ec45000-96ef-4f9c-8910-2cd89ced4711.png)

QA Notes:
Start on Detroit Partner's portal.
https://detroit-partners-dev.netlify.app/

1. Create a new listing
2. Add a unit group and verify that the Accessibility section has been removed from the drawer.
3. Scroll down to the 'Community Types' section and interact with it thoroughly to ensure that there is consistent use of the 'Community Types' over 'Community Programs'.
4. Scroll down to see 'Building Features' and ensure that there is text input labeled 'Additional Accessibility Details'
5. Navigate to the 'Application Process'
6. Verify that under the 'Leasing Agent Section', its first text input is titled 'Property Management Company'.
7. Scroll down further to see that 'Is there a referral opportunity?' and 'Additional Application Submission Notes' inputs have been removed.
8. Save Listing.
9. Ensure consistency on wording changes in Summary as well.

Then go to the public site.
https://detroit-public-dev.netlify.app/listings

1. Navigate to a listing with both Community Types and Additional Accessibility details added and click into the detailed view.
2. Ensure that the new wording of 'Additional Accessibility Details' and 'Community Types' is present in all languages.
